exasol_script_languages_container_tool/lib/tasks/upload/upload_container_base_task.py

```diff
@@ -93,7 +93,7 @@ def _get_upload_url(self, release_info: ExportInfo, without_login: bool = False)
             login = ""
         else:
             login = f"""{self.bucketfs_username}:{self.bucketfs_password}@"""
-        path_in_bucket = f"{self.path_in_bucket}/" if self.path_in_bucket is not None else ""
+        path_in_bucket = f"{self.path_in_bucket}/" if self.path_in_bucket not in [None, ""] else ""
```

Comment:
did you try this in [None,""]

Reply:
yes, tried in python console with:
1. `None in [None, ""]` => true
2. '"" in [None, ""]` => true
3. `"abc" in [None, ""] => false